### PR TITLE
upgrading dind-aws version (docker/docker-compose)

### PR DESCRIPTION
**Pull request by saugereau**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ matrix:
   include:
     - env: LANGUAGE=ansible       VERSION=2.3.0.0-r1 PYTHON_VERSION=2.7.14-r0
     - env: LANGUAGE=aws           VERSION=1
-    - env: LANGUAGE=dind-aws      VERSION=1    DOCKER_COMPOSE_VERSION=1.21.2 GLIBC_VERSION=2.23-r3
+    - env: LANGUAGE=dind-aws      VERSION=1    DOCKER_COMPOSE_VERSION=1.22.0 GLIBC_VERSION=2.23-r3
```
Comment (mremi) [inline .travis.yml]: Can you update the CHANGELOG.md please?

Comment (saugereau) [reply .travis.yml]: Fixed, but I see that ansible build always fail, even in previous merge ...is it normal ?

Comment (mremi) [reply .travis.yml]: Yep please replace 08 by 09, and about the failing build, it's not normal :), can you have a look please?

Comment (saugereau) [reply .travis.yml]: @mremi fixed by replacing python version in ansible build

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 Versions
 ========
 
+2018-09-DEV
+-----------
+* Update dind-aws to use 18.06.1-ce-dind
```
[the remark this answers is not in the document]
Comment (saugereau) [reply CHANGELOG.md]: fixed

**Pull request merged**